webclient/src/components/ResetPasswordDialog/ChangePasswordDialog.tsx

```diff
@@ -10,7 +10,7 @@ import { ResetPasswordForm } from 'forms';
 
 import './ResetPasswordDialog.css';
 
-const ResetPasswordDialog = ({ classes, handleClose, isOpen, onSubmit }: any) => {
+const ChangePasswordDialog = ({ classes, handleClose, isOpen, onSubmit }: any) => {
```

Comment:
I would prefer we keep calling this ResetPassword, ChangePassword will be a different bit of functionality that will require different fields and endpoints.